### PR TITLE
Better example for BulkInsert

### PR DESCRIPTION
**Pull request by froemken**

### CODE REVIEW

File: Documentation/ApiOverview/Database/Connection/Index.rst [hunk 1]
```diff
@@ -95,19 +95,28 @@ bulkInsert()
...
-      ]
+       'sys_log',
+       [
+           [
+               (int)$userId,
+               (string)$details1
+           ],
+           [
+               'userId' => (int)$userId,
+               'detail' => (string)$details2
```
Comment (linawolf) [inline Documentation/ApiOverview/Database/Connection/Index.rst]: If I understand you correctly you are trying to show two alternative Strategies of mapping the data to the fields here? That is why array one and array two of the second parameter look so different?

I would suggest to break up the one example into two examples and some explanatory lines of what you are trying to achieve

File: Documentation/ApiOverview/Database/Connection/Index.rst [hunk 2]
```diff
@@ -95,19 +95,28 @@ bulkInsert()
...
+           ],
+       ],
+       [
+           // 'userid' does not exist in both records, so Doctrine uses value at array key position 0 of data parameter.
```
Comment (linawolf) [inline Documentation/ApiOverview/Database/Connection/Index.rst]: i don't really get what this comment is trying to convey? You mean that there is one input array lacking the key 'userid' ?

**Comment by tmotyl** [comment]
@froemken will you please update your PR?

**Comment by froemken** [comment]
No, I have deleted my repository a few weeks ago.

**Pull request closed without merging**